mysql-test/suite/innodb/t/temp_table_encrypt.test

```diff
@@ -4,6 +4,18 @@
 call mtr.add_suppression("\\[Error\\] InnoDB: keyring error: please check that a keyring plugin is loaded");
 call mtr.add_suppression("\\[Error\\] InnoDB: Can't set temporary tablespace to be encrypted because keyring plugin is not available");
 
+--echo #
+--echo # PS-4727: instrinsic temp table behaviour shouldn't depend on innodb_encrypt_tables
+--echo #
+CREATE TABLE t1 (a INT, b BLOB);
+INSERT INTO t1 VALUES (1, 'hello'), (2, 'hi'), (3, 'satya');
+SET GLOBAL innodb_encrypt_tables=ON;
+SET big_tables=ON;
+INSERT INTO t1 SELECT * FROM t1;
+DROP TABLE t1;
+SET big_tables=default;
```

Comment:
Instead of manipulating big_tables, a shorter version would be SELECT SQL_BIG_RESULT ...